e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -186,6 +188,20 @@ private String verifyJsonFromStatusResponse(HttpResponse<String> statusResponse,
         String type = outputObject.getString("type");
         Assert.assertEquals(type, "ExplanationOfBenefit");
 
+        JSONArray extension = outputObject.getJSONArray("extension");
+        JSONObject checkSumObject = extension.getJSONObject(0);
+        String checkSumUrl = checkSumObject.getString("url");
+        Assert.assertEquals("https://ab2d.cms.gov/checksum", checkSumUrl);
+        String checkSum = checkSumObject.getString("valueString");
+        Assert.assertEquals("sha256:", checkSum.substring(0, 7));
+        Assert.assertEquals(checkSum.length(), 71);
+
+        JSONObject lengthObject = extension.getJSONObject(1);
+        String lengthUrl = lengthObject.getString("url");
+        Assert.assertEquals("https://ab2d.cms.gov/file_length", lengthUrl);
+        long length = lengthObject.getLong("valueDecimal");
+        Assert.assertThat(length, greaterThan(0L));
```

Comment:
Can we take this a step further? Let's have the e2e test actually download the files and _verify_ that the checkum & length match.

Reply:
Yeah I can return the data from this method and then verify it in the later step that downloads the file.